src/retyped/jbuild

```diff
@@ -2,4 +2,4 @@
 
 (library
   ((name retyped)
-    (libraries (sedlex flow_parser))))
+    (libraries (js_of_ocaml sedlex flow_parser))))
```

Comment:
This is to get access to `Regexp` in jsoo. It's already used in `src`, so this should be fine, right?